glade_forms/lib/src/core/glade_input.dart

```diff
@@ -331,7 +333,7 @@ class GladeInput<T> {
     required int value,
     String? inputKey,
     int? initialValue,
-    ValidatorFactory<int>? validator,
+    IntValidatorFactory? validator,
```

Comment:
This is wrong. 

General GladeInput  cant have Int* specialized factory. 

Add GladeInput.int() factory constructor

Reply:
Sorry my bad. I overlooked which static method it is